src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -320,24 +338,28 @@ uiModules.get('apps/management')
 
     let lastPromise;
     resetIndex();
-    samplePromise = lastPromise = updateSamples()
+    samplePromise = lastPromise = Promise.resolve()
```

Comment:
I think it'd be clearer if we use the pattern you linked to. I find this:

```javascript
samplePromise = lastPromise = Promise.resolve()
```

to be a real head scratcher.